scala-generator/src/main/scala/models/Play2Bindables.scala

```diff
@@ -12,6 +12,7 @@ case class Play2Bindables(ssd: ScalaService) {
       "object Bindables {",
       Seq(
         "import play.api.mvc.{PathBindable, QueryStringBindable}",
+        "import play.api.routing.sird.PathBindableExtractor",
```

Comment:
I added `PathBindableExtractor`, they are similar to the other `PathBindable`, and `QueryStringBindable`, but work on the `sird` router.